ui-tests/cypress/integration/administration/email-configuration-Sender.spec.js

```diff
@@ -0,0 +1,306 @@
+import HomePage       from '../../support/pageObjects/HomePage';
+import {htmlElements} from '../../support/pageObjects/WebElement';
+
+describe('Sender Management Functionalities', () =>{
+    let testCode = 'TestCode';
+      let testEmail = 'test@entando.com';
+      let defaultCode1 = 'CODE1';
+      let defaultEmail1 = 'EMAIL1@EMAIL.COM';
+      let defaultCode2 = 'CODE2';
+      let defaultEmail2 = 'EMAIL2@EMAIL.COM';
+      let currentPage;
+
+
+    beforeEach(() =>{
+       cy.kcLogin('login/admin').as('tokens');
+       currentPage = openEmailConfigurationPage();
+       currentPage = openSenderPage();
+    });
+
+    afterEach(() => {
+        
+        cy.kcLogout();
+      });
+
+    describe([Tag.SMOKE, 'ENG-3299'], 'Sender Current Configuration is displayed', () => {
+        
+
+      it([Tag.SMOKE, 'ENG-3299'], 'Table and Button are visible', () => {
+        
+
+        cy.validateAppBuilderUrlPathname('/email-config/senders');  
+        currentPage.getContent().getSenderTable().should('be.visible');
+        currentPage.getContent().getTabHeaders().children(htmlElements.th)
+                  .should('have.length', 3)
+                  .then(elements => cy.validateListTexts(elements, ['Code', 'Email', 'Actions']));
+        currentPage.getContent().getAddButton()
+                  .should('be.visible')
+                  .and('have.text', 'Add');
+
+      });
+
+      it([Tag.SMOKE, 'ENG-3299'], 'New Sender is displayed', () => {
+
+        currentPage.getContent().getAddButton().click().wait(1000);
+        cy.validateAppBuilderUrlPathname('/email-config/senders/add');
+        currentPage.getContent().getSenderForm()
+                   .should('be.visible')
+                   .and('contain', 'Code ')
+                   .and('contain', 'Email ');
+
+        currentPage.getContent()
+                   .getCodeInput()
+                   .should('be.visible')
+                   
+        currentPage.getContent()
+                   .getEmailInput()
+                   .should('be.visible');
+
+      });
+      it([Tag.SMOKE, 'ENG-3299'], 'Action Buttons are diplayed', () => {
+         
+
+        currentPage.getContent().getActionButton().click().wait(500);
+
+        currentPage.getContent().getContextMenu().should('be.visible');
+        currentPage.getContent().getContextMenu()
+                    .contains('Edit')
+                    .should('be.visible')
+                    .should('have.text','Edit');
+        currentPage.getContent().getContextMenu()
+                    .contains('Delete')
+                    .should('be.visible')
+                    .should('have.text','Delete');
+
+
+      });
+      it([Tag.SMOKE, 'ENG-3299'], 'Edit Sender is displayed', () => {
+
+
+        currentPage.getContent().getActionButton().click();
+        currentPage.getContent().getEditButton().click().wait(1000);
```

Comment:
Please, add a comment to explain why the wait is needed and, if it's always needed when doing this _click_, move it to the page object